ng-client/package.json

```diff
@@ -34,7 +34,7 @@
     "@angular/cli": "1.0.0",
     "@angular/compiler": "4.0.2",
     "@angular/compiler-cli": "4.0.2",
-    "@types/jasmine": "2.5.38",
+    "@types/jasmine": "2.5.45",
```

Comment:
Note that this has to be `2.5.45`, because the latest version has an error where it is not aware that protractor's expect function can handle async/promise values (e.g. `element.getText()` returns a promise)